client/components/domains/register-domain-step/index.jsx

```diff
@@ -445,7 +445,7 @@ class RegisterDomainStep extends React.Component {
 				<StickyPanel className={ searchBoxClassName }>
 					<CompactCard className="register-domain-step__search-card">
 						<Search
-							additionalClasses={ this.state.clickedExampleSuggestion ? 'is-refocused' : undefined }
+							className={ this.state.clickedExampleSuggestion ? 'is-refocused' : undefined }
```

Comment:
There seems to be a `initialValue` prop that the search package doesn't support.

Also, we're passing a `ref`, but we also create new refs in our component. Could that cause a regression?

Reply:
> Also, we're passing a ref, but we also create new refs in our component. Could that cause a regression?

What do you mean by this? The ref that is passed get's handled by `useImperativeHandle` and duplicates the methods that were being used on the old component by the contexts where refs were being passed (mostly `focus` to be honest).

Reply:
Scratch that, I don't think it's a concern anyway.